docs/pingonefraudIntegration.md

```diff
@@ -0,0 +1,71 @@
+# Using PingOne Fraud Adapter with the Authentication Widget
+
+Depending on your adapter configuration, you might need to take additional steps to use the Authentication Widget.
+
+### System requirements and dependencies
+
+* PingOne Fraud Management Integration Kit 1.0 or later
+
+### Setup
+
+1\. Add the following to your application's sign-on page.
+* Copy the App ID, Client Secret, and Data collection endpoint URL from the PingOne Fraud dashboard and insert them for _<APP_ID>_, _<APP_SECRET>_, and _<ST_URL>_
+
+```javascript
+<!-- Enable below script integrating with Fraud IK -->
+  <script id="stPingId" src="https://assets.pingone.com/p1f/web-sdk/4.2.0/p1f-sdk.js?appId=<APP_ID>"
+  crossorigin="anonymous" defer>
+  </script>
+
+  <script>
+     
+        function onSecuredTouchReady(callback) {
+            if (window["_securedTouchReady"]) {
+                callback()
+            } else {
+                document.addEventListener("SecuredTouchReadyEvent", callback)
+            }
+        }
+
+        function securedTouchInitLoad(userId, appSessionId) {
+            onSecuredTouchReady(function() {
+                _securedTouch.init({
+	              url: "<ST_URL>",
+                    appId: "<APP_ID>",
+                    appSecret: "<APP_SECRET>",
+                    userId: userId,
+                    sessionId: appSessionId,
+                    isDebugMode: false,
+                    isSingleDomain: false,
+                }).then(function() {
+                    console.log("SecuredTouchSDK initialized successfully for sessionId: " + appSessionId);
+                }).catch(function(e) {
+                    console.error("An error occurred. Please check your init configuration", e)
+                });         
+            });
+        }
+```
+2\. Where your web application initializes `PfAuthnWidget`, adjust _<FRAUD_CLIENT_PLATFORM>_ and _<FRAUD_CLIENT_VERSION>_ as appropriate to the application. The _<FRAUD_CLIENT_SESSION_ID>_ should be the unique identifier for the session.
+```javascript
+var authnWidget = new PfAuthnWidget('https://localhost:9031', { divId: 'authnwidget', fraudClientPlatform: <FRAUD_CLIENT_PLATFORM>, fraudClientVersion: <FRAUD_CLIENT_VERSION>, fraudClientSessionID: <FRAUD_CLIENT_SESSION_ID>});
```

Comment:
Is there any way to split this  into multiple lines? You have to horizontally scroll to see it all.